docs/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>mesh orange</title>
+  </head>
+  <body>
+    <code>
+      <p>build <a href="build.pdf">.pdf</a> <a href="build.dot">.dot</a></p>
+      <img src="build.svg" width="1200" />
+      <p>kernel <a href="kernel.pdf">.pdf</a> <a href="kernel.dot">.dot</a></p>
+      <img src="kernel.svg" width="400" />
+      <p>debian minimal builder <a href="debian_minimal_builder.pdf">.pdf</a> <a href="debian_minimal_builder.dot">.dot</a></p>
+      <img src="debian_minimal_builder.svg" width="800" />
+      <p>boot-1 <a href="boot-1.pdf">.pdf</a> <a href="boot-1.dot">.dot</a></p>
+      <img src="boot-1.svg" width="1200" />
+      <p>boot-2 <a href="boot-2.pdf">.pdf</a> <a href="boot-2.dot">.dot</a></p>
+      <img src="boot-2.svg" width="1200" />
+    </code>
+  </body>
+</html>
```

Comment:
The index to the diagrams should be autogenerated from the list of generated svg files - probably using a template and probable by the Makefile